draft-ietf-core-cf-reg-update.md

```diff
@@ -82,13 +81,13 @@ This document uses the terms "media type", "content coding", "content-type", and
 
 # Examples for Erroneous Registrations
 
-This section provides examples of registration requests for the "CoAP Content-Formats" Registry (as defined in {{Section 12.3 of -coap}} and revised according to {{Err4954}}) that are invalid but would be approved under the current procedure.
+This section provides examples of registration requests for the "CoAP Content-Formats" Registry that are invalid but would be approved under the procedure defined in {{Section 12.3 of -coap}}.
 The checklist defined in {{checks}} should prevent any of these attempts from succeeding.
 
 All the example registration requests use a CoAP Content-Format with identifier 64999 in the FCFS range of the "CoAP Content-Formats" registry.
 
-For each of the following example registration requests, one can create a similar instance where the requested registration is for a CoAP Content-Format identifier within the "IETF Review" or "IESG Approval" range of the registry.
-Similarly, such registrations must not be allowed to succeed.
+For each of the following example registration requests, one can create a similar instance where the requested registration is for a CoAP Content-Format identifier within the "IETF Review" or "IESG Approval" ranges.
```

Comment:
Did we mean this actually:

```suggestion
For each of the following example registration requests, one can create a similar instance where the requested registration is for a CoAP Content-Format identifier within the "IETF Review or IESG Approval" range.
```

Or did we want to also include the "Expert Review" range here?

Not sure what the benefit is of mentioning the other ranges though. The problem was in the FCFS range , right?